casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -508,6 +508,13 @@ object ProtoUtil {
       phloLimit = phlos
     )
 
+  def sourceDeployNowWithPhlo(source: String, phlos: Long): DeployData =
+    sourceDeploy(
+      source,
+      System.currentTimeMillis(),
```

Comment:
I prefer if we could use `Time[F].currentMillis` so we can inject logicalTime for tests